config/environments/test.rb

```diff
@@ -46,4 +46,6 @@
   config.active_support.test_order = :random
 
   config.middleware.use NoAnimations
+
+  config.active_job.queue_adapter = :inline
```

Comment:
why you added to process inline

Reply:
In the older version this was the default behaviour, now we must specify that it must behave this way on the test enviroment in order to keep it working the same way